build/do.rq

```diff
@@ -163,6 +182,28 @@ write_readme {
 	run("./regal table --write-to-readme bundle")
 }
 
+fetch_engine_caps {
+	fetch_eopa_caps
+}
+
+fetch_eopa_caps {
+	templ := {
+		"remote": "https://github.com/StyraInc/enterprise-opa.git",
+		"ref": "main",
+		"srcdir": "capabilities",
+		"destdir": eopa_caps_dir
+	}
+
+	# NOTE(charles): My git config forces git to use ssh instead of http
+	# URLs, which is not desired in this case. It follows others my have
+	# similar setups locally. Setting GIT_CONFIG=/dev/null in this case
+	# prevents this.
+
+	fetch_cmd := rq.template("sh -c 'GIT_CONFIG=/dev/null git archive --remote={{.remote | quote}} {{.ref}} {{.srcdir | quote}} | tar -x -C {{.destdir | quote}}'", templ)
```

Comment:
No idea what's preferable, but if you'd consider it less of a hassle, [you could grab the zip](https://docs.github.com/en/repositories/working-with-files/using-files/downloading-source-code-archives):
https://github.com/StyraInc/enterprise-opa/archive/refs/heads/main.zip

Reply:
I was trying to find a good way to only have to download the subset of the repo that is needed, rather than the entire thing. If all else fails, I'll just do that, and/or clone the whole repo into `/tmp` or something.

I don't know _why_ the `git archive` thing doesn't work. StackOverflow says it should. The man page leads me to believe it should. But `run()` is complaining about a mismatched `'`, and on top of that when I run it manually, `git` complains about trying to clone a `git://` URL with the `ssh://` protocol, which is just bizarre.